cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,97 @@
+describe('Flow Tests', () => {
+  it('Shows the landing page', () => {
+    cy.visit('/')
+    cy.get('[data-testid=get_started_button]').click()
+  })
+
+  it('Navigates through the Medicaid only flow', () => {
+    const incomeName = "Suzy"
+    const incomeDescription = "Yardwork"
+    const incomeAmount = '55'
+    const expenseName = 'Gas'
+    const expenseDate = '09/04/2024'
+    const expenseAmount = '33'
+    const username = 'Jane Doe'
+    cy.visit('/')
+
+    // Landing Page
```

Comment:
appreciate you segmenting these out into each part. if these are reliable and we decide we want to invest, we'll want to have a behavior lib here. having the comments here help us to see the patterns.